php/README.md

```diff
@@ -1,11 +1,36 @@
 # PHP CLI on Docker
 
-This is a basic Docker environment for PHP development, based on [debian:jessie](https://hub.docker.com/_/debian/).
+[![Build Status](https://travis-ci.org/akeneo/Dockerfiles.svg?branch=php-7.1)](https://travis-ci.org/akeneo/Dockerfiles/tree/php-7.1)
 
-It is not intended to use directly as it stands, but rather to extend it and create custom development environments.
+This is a basic Docker environment for PHP development and testing, based on [debian:stretch-slim](https://hub.docker.com/_/debian/). **This image does not contain Akeneo PIM**.
 
-It comes with PHP 7.1 (provided by dotdeb.org repository), and some PHP extensions: apcu, mcrypt, intl, mysql, curl, gd,
-mongo, soap, xml, zip and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
+The environment comes with Debian 9 (Stretch) and PHP 7.1 (from [Sury repository](https://deb.sury.org/)), and some PHP extensions: apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
+
+## How to use it?
```

Comment:
if you use "?" then it is a question, which must contain a verb. Otherwise, get rid of it ;)